README.md

```diff
@@ -1,2 +1,98 @@
 # saritasa-github-actions
 GitHub actions
+
+## CI-wordpress reusable workflow
+
+### Inputs variables
+
+| Variable       | Type          | Example                  | Discription                                                     |
+| ---------------|:-------------:| :------------------------|-----------------------------------------------------------------|
+| server_host    | string        | 59.128.16.24             | The address of the server for deploy                            |
+| environment    | string        | production               | The environment for deploy i.e development, staging, production |
+| playbook_path  | string        | ci/ansible/playbook.yaml | The path to the file relative to the repository, used in Ansistrano deploy task |
+| deploy_path    | string        | ~/deploy/                | The path to deploy on the remote server |
+| deploy_repo    | string        | git@github.com:saritasa-nest/ceai-wordpress.git | The path to deploy on the remote server |
+| deploy_branch  | string        | develop | The branch from which the deployment is made  |
+| python_version | number        | 3.11.4 | Each python version supports a certain range of ansible versions, so we need to specify the current version of python in order to install the latest version of ansible. |
+| runner         | string        | ubuntu-latest | The runner on which workflow is running: ubuntu, macos, windows, etc |
+
+### Inputs secrets
+
+| Variable       | Discription                                                    |
+| ---------------|:---------------------------------------------------------------|
+| ssh_private_key | The private key for connect to production server via ssh      |
+
+### Examples
```

Comment:
Can you add here a link to PR to some project, which already uses this workflow

Reply:
done